ksqldb-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -340,6 +340,12 @@
       + "error messages (per query) to hold in the internal query errors queue and display"
       + "in the query description when executing the `EXPLAIN <query>` command.";
 
+  public static final String KSQL_QUERY_STATUS_RUNNING_THRESHOLD_SECS =
+      "ksql.query.status.running.threshold.seconds";
+  private static final Integer KSQL_QUERY_STATUS_RUNNING_THRESHOLD_SECS_DEFAULT = 18000;
```

Comment:
this seems a little high - can we set it to `2700` instead? (3x the max query backoff)

Reply:
I miss typed the value here. It should be 300 (5-min) which is mentioned in Rohan's design.